### ISSUE TITLE
`css-modules-loader-core` is outdated

### ISSUE THREAD

**Issue by yyx990803** [css-modules/postcss-modules#105]
This package delegates a large amount of functionality to [`css-modules-loader-core`](https://github.com/css-modules/css-modules-loader-core), which is very outdated as of now:

- Dependency on postcss v6 (while this package uses v7, so every install results in multiple postcss verisons and their deps)
- [Fixed and outdated versions of `postcss-modules-x` plugins](https://github.com/css-modules/css-modules-loader-core/blob/master/package.json#L9-L16)

I saw a major version bump of this package just two weeks ago, but I think `css-modules-loader-core` really needs an update if this package is meant to be the recommended way of using CSS Modules with postcss.

AFAIK, webpack's `css-loader` pretty much rolls their own css modules postcss plugin that sidesteps this problem.

**Comment by outpunk** [comment]
Agreed, we need to do something with this.
@geelen @markdalgleish any plans on css-modules-loader-core and the plugins?

**Comment by allocenx** [comment]
Hey, the last major version bump, was mostly due to my PR with ** localsConvention** PR. Will try to upgrade those ones as well and have another PR. Thanks.

**Comment by lukeed** [comment]
Solved it here: https://github.com/css-modules/css-modules-loader-core/pull/236

Waiting on feedback. Worst case scenario, I publish and maintain and fork that `postcss-modules` uses.

As it stands, _any_ `postcss-modules` installation brings 3 copies of `postcss` with it 🤦

**Comment by outpunk** [comment]
@lukeed @yyx990803 fixed and released in 3.0.0. Thank you guys for pinging me

**Comment by yyx990803** [comment]
Thanks for taking care of this @lukeed @outpunk !
